components/powersensor/ina/ina.go

```diff
@@ -216,7 +211,7 @@ func (d *ina) calibrate() error {
 	}
 
 	// setting config to 111 sets to normal operating mode
-	err = handle.WriteRegU16BE(configRegister, uint16(0x6F))
+	err = handle.WriteRegU16BE(configRegister, uint16(0x399F))
```

Comment:
Change the comment to match: we're no longer setting it to 111. This is setting it to all the default values, right?